rbc/src/main/java/com/library/rbc/model/Notification.java

```diff
@@ -0,0 +1,27 @@
+package com.library.rbc.model;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import org.springframework.data.annotation.Id;
+import org.springframework.data.mongodb.core.mapping.Document;
+
+import java.time.LocalDateTime;
+
+@Document(collection = "notifications")
+@Data
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
+public class Notification {
+
+    @Id
+    private String id;
+    private Boolean isRead;
+    private String subject;
+    private String content;
+    private LocalDateTime creationDateTime;
```

Comment:
We can go with `creationDate`, like in the `Suggestion` class.

Reply:
Would it be better to keep the name creationDateTime, because notification itself contains informations about the time?

Reply:
In my experience, `creationDate` is the commonly used name for this purpose. Alternatively, you could go with `creationTime`, then the field name should also be changed in the `Suggestion` class, and any other classes that have similar fields.